cli/compat/mod.rs

```diff
@@ -23,18 +23,18 @@ static SUPPORTED_MODULES: &[&str] = &[
   "async_hooks",
   "buffer",
   "child_process",
-  "cluster",
+  // "cluster",
```

Comment:
@bartlomieju the commented modules are actually not resolved and throws Module Not found error, how do we want to handle this ?